mautic/mautic
# Report email stats enhacement

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Follow up to https://github.com/mautic/mautic/pull/7214


This PR change logic to generate email stats (Data Source: Email).

Emails stats before were load from summarize data.
Now If you **Hide date range** is disabled it respect date range and show numbers of sent, read, unsubscribed, bounced, clicked related to date range.

If you **Hide date range** is enabled, then show global summarize data for all time.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com/7214) or load to you local enviroment with stats
2. Jus retest both case
- If Hide date range is enabled/disabled
![image](https://user-images.githubusercontent.com/462477/52280292-dee7b280-295b-11e9-9622-23c76c103e95.png)
- If enabled - then try load custom date range and see If stats are related and what do you expect from that range
If disabled - see global stats to each email
Also try filters for both state

![image](https://user-images.githubusercontent.com/462477/52279816-b8754780-295a-11e9-84d4-a595c3bad580.png)



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10767"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
